docs/subaccount/how-to-create-a-integration-with-a-subaccount.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: how-to-create-a-integration-with-a-subaccount
+title: Como criar uma integração vinculada à uma subconta
+tags:
+  - api
+  - subaccount
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info
+Para a utilização desta funcionalidade é necessário possuir a funcionalidade Subconta
+:::
+
+Existem algumas situações em que você precisa destinar todo o montante de uma aplicação para uma subconta. Em vez de precisar fazer vários splits ou especificar a subconta de destino nas requisições, você pode criar uma aplicação que fará isso automaticamente para você.
+
+Para isso, basta seguir estes passos:
+
+1. Acesse a seção API / Plugins.
```

Comment:
Each step of the tutorial should be a heading instead.

It's easier to link to each step because it turns a hash that the user can easily redirect to.